parm/ufs/chem/ExtData.other

```diff
@@ -135,16 +132,16 @@ BRC_AVIATION_CRS NA  Y Y %y4-%m2-%d2t12:00:00 none none oc_aviation /dev/null
 # SOA production
 pSOA_BIOB_VOC NA Y Y %y4-%m2-%d2t12:00:00 none none biofuel /dev/null
 
-# ======= Nitrate Sources ========
-EMI_NH3_AG   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_ag      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
-EMI_NH3_EN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_en      /dev/null
-EMI_NH3_IN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_in      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
-EMI_NH3_RE   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_re      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
-EMI_NH3_TR   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_tr      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
-EMI_NH3_OC   'kg m-2 s-1'  Y Y      %y4-%m2-%d2T12:00:00    none     none     emiss_ocn   ExtData/PIESA/sfc/GEIA.emis_NH3.ocean.x576_y361.t12.20080715_12z.nc4
-
-# --------------------------------------------------------------
-# If using 64 levels please replace this section with the correct values (ie replace 127 with 64)
-NITRATE_HNO3 'mol mol-1'   Y N      %y4-%m2-%d2T12:00:00    none     0.20     hno3        ExtData/PIESA/L127/GMI.vmr_HNO3.x144_y91.t12.2006.nc4
-# --------------------------------------------------------------
-NI_regionMask NA           Y V      -                       none     none     REGION_MASK ExtData/PIESA/sfc/ARCTAS.region_mask.x540_y361.2008.nc
+# # ======= Nitrate Sources ========
+# EMI_NH3_AG   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_ag      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_EN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_en      /dev/null
+# EMI_NH3_IN   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_in      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_RE   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_re      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_TR   'kg m-2 s-1'  N Y      %y4-%m2-%d2T12:00:00    none     none     NH3_tr      ExtData/nexus/CEDS/v2019/%y4/CEDS.2019.emis.%y4%m2%d2.nc
+# EMI_NH3_OC   'kg m-2 s-1'  Y Y      %y4-%m2-%d2T12:00:00    none     none     emiss_ocn   ExtData/PIESA/sfc/GEIA.emis_NH3.ocean.x576_y361.t12.20080715_12z.nc4
+
+# # --------------------------------------------------------------
+# # If using 64 levels please replace this section with the correct values (ie replace 127 with 64)
+# NITRATE_HNO3 'mol mol-1'   Y N      %y4-%m2-%d2T12:00:00    none     0.20     hno3        ExtData/PIESA/L127/GMI.vmr_HNO3.x144_y91.t12.2006.nc4
+# # --------------------------------------------------------------
+# NI_regionMask NA           Y V      -                       none     none     REGION_MASK ExtData/PIESA/sfc/ARCTAS.region_mask.x540_y361.2008.nc
```

Comment:
Unless this is a temporary change, I'd prefer all these blocks of commented out fields to just be removed to keep things clean.

Reply:
I understand but I'm hesitant to remove them because future applications may use them.  @rmontuoro

Reply:
I echo @bbakernoaa and appreciate the desire to keep things cleaned and just simply remove, but from my understanding I think this might be a case where it'll help more people to leave it in.    While the configuration will be without nitrates for now, it seems very likely this is something that would be turned on for testing in the near future.  I'll of course defer to the aerosols group on this though in terms of how frequently they think this would be turned on/off and the need to leave this here.  It seems like the commented out diag table, and feild table would all fall under this.

Reply:
@JessicaMeixner-NOAA I concur. It would be helpful to leave it in for further development and testing rather than completely remove it.  Many contributors wouldn't know it was possible then.